src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/model/APIListing.java

```diff
@@ -13,7 +13,10 @@
     private List<ChildItem> childItems;
 
     @JsonProperty("Name")
-    private String Name;
+    private String name;
+
+    @JsonProperty("Language")
+    private String language;
```

Comment:
This is a good change but we need to double-check that it works as expected. The code around language detection is APIView a bit of a mess.

Reply:
Yeah - I figured it would need to be wired up appropriately. I'm happy to have this PR sit here until it is ready to be supported. I imagine a day where we can do what I said above - have sub-sites for each language - but also have a way of tying together an APIView for all languages on a particular client library.

Reply:
@pakrym Let me know when this PR should be merged.

Reply:
This week is super busy for me. @praveenkuttappan can you deploy this change to a staging instance and verify that things are working well?

Reply:
There's no rush on my end either - we can sit on this for as long as possible, or I can remove the Language property from the json and just merge in the other Java-specific improvements.

Reply:
I will try to deploy to staging and we can verify it on it before we merge.

Reply:
I have deployed your changes here. https://prmarott-apiviewtest.azurewebsites.net/ to test . I have done quick testing with uploads and revisions and looks good.

I will rename this instance to staging and make a pipeline in internal to help anyone triggering deployment.